lib/class-wp-rest-menu-items-controller.php

```diff
@@ -623,7 +691,7 @@ protected function prepare_links( $menu_item ) {
 					'embeddable' => true,
 				);
 			}
-		}
+		}//end if
```

Comment:
Ditto.

Reply:
Fixed in https://github.com/WP-API/menus-endpoints/pull/41/commits/071f64f3c82084544f6050f636b081eae460b344